skyportal/handlers/api/followup_apis.py

```diff
@@ -0,0 +1,16 @@
+from baselayer.app.access import auth_or_token
+from skyportal import facility_apis
+from skyportal.enum_types import api_classnames
+
+from ..base import BaseHandler
+
+
+class FollowupAPIsHandler(BaseHandler):
```

Comment:
Why don't we just compute this once outside of the handler and then just return it? I mean it shouldn't change while the app is running right?